datascience/util.py

```diff
@@ -22,3 +24,30 @@ def percentile(p, arr=None):
     if arr is None:
         return lambda arr: percentile(p, arr)
     return np.percentile(arr, p, interpolation='higher')
+
+def plotCDFarea(rbound, lbound=None, mean=0, sd=1):
```

Comment:
I believe we use snake_case for our function names. So this would be called `plot_cdf_area`.